pkg/codegen/docs/gen.go

```diff
@@ -1129,10 +1132,20 @@ func (mod *modContext) genResource(r *schema.Resource) resourceDocArgs {
 		Notes:      mod.pkg.Attribution,
 	}
 
+	var titleTag string
+	if val, ok := titleLookup[mod.pkg.Name]; ok {
+		titleTag = val
+	} else {
+		titleTag = mod.pkg.Name
+	}
+
 	stateParam := name + "State"
+
 	data := resourceDocArgs{
 		Header: header{
-			Title: name,
+			Title:    name,
+			TitleTag: fmt.Sprintf("Resource %s | Package %s", name, titleTag),
```

Comment:
Could we get a `Module %s | ` in there as well, so that what you end up with is `Resource Bucket | Module s3 | Package AWS` here?

Reply:
Yea, just noticed this myself when I was adjusting the index pages.... thank you.